status.py

```diff
@@ -227,19 +239,20 @@
                     },
                     "Run_Status": {
                         "tags": [ TAG.Run, TAG.Status ],
+                        "subclasses": {
+                            "Run_Enable_Status": {}, # defined elsewhere
```

Comment:
Same as one of the previous comments; I'd rather use "parents" annotation for the consistency.

Reply:
Fixed!